Framework/Geometry/inc/MantidGeometry/Rendering/ShapeInfo.h

```diff
@@ -0,0 +1,96 @@
+#ifndef MANTID_GEOMETRY_SHAPEINFO_H_
+#define MANTID_GEOMETRY_SHAPEINFO_H_
+
+#include "MantidGeometry/DllConfig.h"
+#include <vector>
+
+namespace Mantid {
+namespace Kernel {
+class V3D;
+}
+namespace Geometry {
+class RectangularDetector;
+class StructuredDetector;
+class IObjComponent;
+
+/** ShapeInfo : Stored Object related shape information for rendering.
+
+Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge
+National Laboratory & European Spallation Source
+
+This file is part of Mantid.
+
+Mantid is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 3 of the License, or
+(at your option) any later version.
+
+Mantid is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+File change history is stored at: <https://github.com/mantidproject/mantid>
+Code Documentation is available at: <http://doxygen.mantidproject.org>
+*/
+namespace detail {
+class MANTID_GEOMETRY_DLL ShapeInfo {
+public:
+  enum class GeometryShape {
+    NOSHAPE = 0,
+    CUBOID,            ///< CUBOID
+    HEXAHEDRON,        ///< HEXAHEDRON
+    SPHERE,            ///< SPHERE
+    CYLINDER,          ///< CYLINDER
+    CONE,              ///< CONE
+    SEGMENTED_CYLINDER ///< Cylinder with 1 or more segments (along the axis).
```

Comment:
I believe `SEGMENTED_CYLINDER` is never used. Could you remove it please?

Reply:
It's used here https://github.com/mantidproject/mantid/blob/143ecb071b1db696d625c50b141e6a3e5e1bbbcf/Framework/Geometry/src/Rendering/Renderer.cpp#L252

Reply:
I don't think any instrument creates a `SEGMENTED_CYLINDER`

Reply:
But should we introduce an instrument which requires this we will have to replace it.